scripts/lua/rest/v1/get/network/alert/list.lua

```diff
@@ -40,13 +40,25 @@ interface.select(ifid)
 -- Fetch the results
 local alerts, recordsFiltered = network_alert_store:select_request()
 
-for _key,_value in ipairs(alerts or {}) do
-   local record = network_alert_store:format_record(_value, no_html)
+for _, _value in ipairs(alerts or {}) do
+   local record = {}
+
+   if no_html then
```

Comment:
To avoid duplicating this `if no_html then ... else ... end` logic inside any of the alert endpoints available, just rely on the `no_html` parameter already existing. You can do a single `alert_store:format_record(_value, no_html)` and put the `if-then-else` only one time inside that function. Even better, you can do a single wrapper for this logic and also to decide between `vanilla_payload_response` and `extended_answer`. This will save a lot of code lines.